wfl/Controllers/MenuSelectionsController.cs

```diff
@@ -24,6 +24,16 @@ public class MenuSelectionsController : ApiController
             return Ok(order.MenuSelections.FirstOrDefault(x => x.Id == selectionId));
         }
 
+        [Route("{selectionId}")]
+        [HttpDelete]
+        public async Task<IHttpActionResult> DeleteByIdAsync([FromUri] string foodOrderId, [FromUri] string selectionId)
+        {
+            FoodOrder order = await dbManager.GetEntityByIdAsync<FoodOrder>(foodOrderId);
+            order.MenuSelections = order.MenuSelections.Where(x => x.Id != selectionId).ToList();
```

Comment:
While this code will work to remove the menu selection in question, I think it'll make sense for us to look at making these changes with more minimal effort.  Even though for this application, we will never have an issue with scalability, it's still a good concept to be aware of.  In this case we could have simply removed a menu selection from the existing collection and saved the same object.  However, instead we are creating an entirely new collection, assigning that new collection to the object and then throwing the old one away.  I would recommend simply removing the item from the list and saving.